sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/OfferAccepter.java

```diff
@@ -26,6 +23,10 @@
 
     private Collection<OperationRecorder> recorders;
 
+    // Metrics
+    private final Counter launchCount = SchedulerUtils.getMetricRegistry().counter("launch");
```

Comment:
May want to dot-delimit these. For example `offer.response.launch`, `offer.response.launch_group`, `offer.response.decline`, ...

Reply:
Having some kind of nesting like this is also great for selecting multiple metrics in Grafana!